string bodyA_name
string bodyB_name
---
geometry_msgs/Pose relpose
bool ok
